### PR TITLE
workflow tidying

### PR DESCRIPTION
**Pull request by redraincatching**
pr split on behalf of @jonesmz 

normalisation of some of the workflow files

### CODE REVIEW

File: .github/workflows/macos.yml [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
-    branches: [ $default-branch ]
   pull_request:
     types: [ opened, reopened, synchronize ]
 
 jobs:
   builds:
-    runs-on: macos-12
+    strategy:
+      fail-fast: false
+        # ubuntu 16.04 EOL is 2026
```
Comment (eakraly) [inline .github/workflows/macos.yml:12]: Copy paste?

File: .github/workflows/macos.yml [hunk 2]
```diff
@@ -1,16 +1,24 @@
...
+      matrix: 
+        os: [ 'macos-11', 'macos-12' ]
```
Comment (eakraly) [inline .github/workflows/macos.yml:14]: Why adding 11?

Comment (redraincatching) [reply .github/workflows/macos.yml:13]: github supports MacOSX, although if you would rather not support it it can be removed

Comment (jonesmz) [reply .github/workflows/macos.yml:13]: IMHO coturn should explicitly declare the platforms supported, and out of that list, any that github has runners for should be used.

If Mac OSX 11 isn't supported, then lets declare that to be the case. Else, take advantage of the equipment offered by github.

Comment (eakraly) [reply .github/workflows/macos.yml:13]: I think one version of mac is enough
Especially if that is older one

**Comment by ggarber** [comment]
I think it could be cleaned up a bit more but it is better than now so LGTM.

As for MacOS versions I don't have a strong opinion so let's see how it goes with both versions.

Probably any decission is better than keeping the PR open longer in this case.

**Pull request merged**